### ISSUE TITLE
腾讯企鹅号改版导致 RSS 失效

### ISSUE THREAD

**Issue by martin5lee98** [DIYgod/RSSHub#12649]
### Routes

```routes
/tencent/news/author/:mid
```

### Full routes

```fullroutes
https://rsshub.app/tencent/news/author/5933889
```

### Related documentation

https://docs.rsshub.app/new-media.html#teng-xun-qi-e-hao-geng-xin

### What is expected?

显示腾讯企鹅号的更新

### What is actually happening?

提示 Error message: Cannot read properties of null (reading 'name')

### Deployment information

RSSHub demo (https://rsshub.app)

### Additional info

```shell
看起来腾讯企鹅号已经做了改版，原有的 URL 以及 RSS 抓取规则已经失效。

举例：腾讯科技的页面 URL，本身是，https://new.qq.com/omn/author/5540552

现在是，

https://new.qq.com/omn/author/8QMZ3XxU64wZuTw%3D

但即使访问 https://rsshub.app/tencent/news/author/8QMZ3XxU64wZuTw%3D 依然会得到同样的报错，

Cannot read properties of null (reading 'name')
```

### This is not a duplicated issue

- [X] I have searched [existing issues](https://github.com/DIYgod/RSSHub/issues) to ensure this bug has not already been reported

**Comment by dev-techmoe** [comment]
似乎问题并没有解决。

目前打开 https://rsshub.app/tencent/news/author/8QMZ3XxU64wZuTw%3D 依然是报错的。

```
Route requested: /news/author/8QMZ3XxU64wZuTw%3D
Error message: Cannot read properties of null (reading 'nick')
Helpful Information to provide when opening issue:
Path: /news/author/8QMZ3XxU64wZuTw%3D
Node version: v18.16.1
Git Hash: 3e8a976
```

**Comment by Jiang10086** [comment]
> 似乎问题并没有解决。
> 
> 目前打开 https://rsshub.app/tencent/news/author/8QMZ3XxU64wZuTw%3D 依然是报错的。
> 
> ```
> Route requested: /news/author/8QMZ3XxU64wZuTw%3D
> Error message: Cannot read properties of null (reading 'nick')
> Helpful Information to provide when opening issue:
> Path: /news/author/8QMZ3XxU64wZuTw%3D
> Node version: v18.16.1
> Git Hash: 3e8a976
> ```

朋友，试试用 https://rsshub.app/tencent/news/author/5540552

**Comment by huanfe1** [comment]
fixed in #16798
